tests/test_event_handling.py

```diff
@@ -24,15 +24,15 @@
 from hexkit.utils import calc_part_size
 
 from tests.fixtures.config import Config
-from tests.fixtures.file_fixtures import (
+from tests.fixtures.joint import (
```

Comment:
This is not testing the multinode ability, i.e. the ability to address two s3 services, right? Might be worth to add since this is the main feature.

Reply:
Should now use the correct config for both test paths